bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -114,6 +122,8 @@ sitemap demo label="Main Menu" {
         Setpoint item="TemporaryRoomSetpointOverride" icon="temperature" label="Temporary room setpoint override [%.1f °C]" minValue="0" maxValue="30" step="0.1"
         Setpoint item="ConstantRoomSetpointOverride" icon="temperature" label="Constant room setpoint override [%.1f °C]" minValue="0" maxValue="30" step="0.1"
         Text item="ControlSetpoint" icon="temperature" label="Control setpoint [%.1f °C]"
+        Text item="ControlSetpointRequested" icon="temperature" label="Control setpoint requested [%.1f °C]"
+        Text item="ControlSetpointOverride" icon="temperature" label="Control setpoint override [%.1f °C]"
```

Comment:
Should `ControlSetpointOverride` be a Setpoint?

Reply:
Yes, it should have been, corrected